### ISSUE TITLE
Reserved keywords in page templates

### ISSUE THREAD

**Issue by Rowanism** [sulu/sulu#4100]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.6.13

#### Actual Behavior

When using reserved keywords as property names in a page template, problems occur when trying to save pages.

#### Example

Using the reserved keyword `id` as a property name results in errors when trying to create a new page:

`Could not find document with ID or path "xyz"`
`RepositoryException: Not an absolute path 'xyz'`

#### Steps to Reproduce

- Create new page template in a webspace
- Create a property with a reserved keyword as name
```
<property name="id" type="text_line">
    <meta>
        <title lang="en">ID</title>
    </meta>
</property>
```
- Create a new page in the webspace
- Save or Publish the page

**Comment by danrot** [comment]
We already have such a list in https://github.com/sulu/sulu/blob/b22ce62f0100bcef9ed4ed697f4b9bef09d6226b/src/Sulu/Component/Content/Metadata/Loader/StructureXmlLoader.php#L56, but `id` seems to be missing (didn't know it was making problems at all...). If you want you can create a PR, which adds this conflicting name.

**Comment by danrot** [comment]
Fixed in #5026
